cluster-scope/base/namespaces/codait-advo/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: codait-advo
```

Comment:
is this to be added to zero cluster?
missing reference here: https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/zero/kustomization.yaml

Reply:
good catch :+1: